chromadb/utils/embedding_functions.py

```diff
@@ -688,7 +688,7 @@ def __init__(
             )
 
         model, _, preprocess = open_clip.create_model_and_transforms(
-            model_name=model_name, pretrained=checkpoint
+            model_name=model_name, pretrained=checkpoint, cache_dir=cache_dir if cache_dir is not None else None
```

Comment:
Will there be a difference if you only use `cache_dir=cache_dir`? If the underlying method handles None, then there is no need to check for it.